config/application.yml.example

```diff
@@ -63,6 +63,7 @@ development:
   aamva_public_key: '123abc'
   aamva_private_key: '123abc'
   aamva_verification_url: 'https://example.org:12345/verification/url'
+  agencies_with_agency_based_uuids: ''
```

Comment:
At first glance, I wasn't expecting this to be a list of numbers. Can we add a comment to explain which numbers correspond to which agency and how to get that information? Ideally, these would be strings with agency names or other easier way to identify the agencies. What was the reasoning behind using integers?

Reply:
The rationale behind using integers is that they can change the names at any time or use long names.  It also simplifies the code a bit.

Reply:
Gotcha. Sounds good.